contents/tutorials/aa-testing.md

```diff
@@ -0,0 +1,92 @@
+---
+title: How to do A/A testing
+date: 2023-07-06
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-4.png
+tags: ['experimentation', 'feature flags']
+---
+
+An A/A test is the same as an [A/B test](/docs/experiments) except both groups getting the same code or component. Because both groups get functionally identical variants, you want to see no statistical difference between the variants by the end of the experiment. If there is a significant difference, that is cause to investigate.
```

Comment:
```suggestion
An A/A test is the same as an [A/B test](/docs/experiments) except both groups receive the same treatment or product experience. Since both groups get identical functionality, the goal is to not see a statistical difference between the variants by the end of the experiment. 
```